integrations-and-supported-tools/detectron2/notebooks/Neptune_detectron2.ipynb

```diff
@@ -0,0 +1,490 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "GG3_2HyRVY4Y",
+    "tags": [
+     "header",
+     "comment"
+    ]
+   },
+   "source": [
+    "# Neptune + Detectron2\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "This guide is adapted from the [official Detectron2 tutorial](https://colab.research.google.com/drive/16jcaJoc6bCFAQ96jDe2HwtXj7BMD_-m5?usp=sharing) to show you,\n",
+    "\n",
+    "* how to integrate neptune with detectron2 training loop\n",
+    "* how to save checkpoints from training stage.\n",
+    "* how to visualize and log the prediction of the trained model\n",
+    "\n",
+    "[See this example in Neptune](https://app.neptune.ai/o/common/org/detectron2-integration/e/DET-3/all)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "FUaDrmRu-h3k"
+   },
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune anonymously, with zero setup.\n",
+    "\n",
+    "* If you're running the notebook on your local machine, you need to have [Python](https://www.python.org/downloads/) and [pip](https://pypi.org/project/pip/) installed.\n",
+    "* If you want to see the example logged to your own workspace instead:\n",
+    "    * Create a Neptune account → [Take me to registration](https://neptune.ai/register)\n",
+    "    * Create a Neptune project that you will use for tracking metadata → [Tell me more about projects](https://docs.neptune.ai/administration/projects)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "w_0KXqlePqNi"
+   },
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "i36OvZAgvxkj"
+   },
+   "outputs": [],
+   "source": [
+    "! pip install -U neptune-client neptune-detectron2 opencv-python"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! python -m pip install pyyaml==5.1\n",
+    "import sys, os, distutils.core\n",
+    "\n",
+    "# Note: This is a faster way to install detectron2 in Colab, but it does not include all functionalities.\n",
```

Comment:
Maybe this comment (lines 72 and 73) would be nicer to have in a proper text cell?